apps/bors_frontend/web/controllers/project_controller.ex

```diff
@@ -89,6 +90,25 @@ defmodule BorsNG.ProjectController do
       update_branches: Project.changeset_branches(project)
   end
 
+  def log(_, :ro, _, _), do: raise BorsNG.PermissionDeniedError
```

Comment:
`:rw` is a superset of `:ro`. I picked the more stringent permission mostly to be conservative.